pkg/pdf/pdf.go

```diff
@@ -358,7 +358,8 @@ func (s *PdfMaroto) FileImage(filePathName string, prop ...props.Rect) {
 	qtdCols := float64(len(s.colsClosures))
 	sumOfyOffsets := s.offsetY + rectProp.Top
 
-	s.Image.AddFromFile(filePathName, sumOfyOffsets, s.rowColCount, qtdCols, s.rowHeight, rectProp)
+	err = s.Image.AddFromFile(filePathName, sumOfyOffsets, s.rowColCount, qtdCols, s.rowHeight, rectProp)
+	return
```

Comment:
Another place where in my opinion it would be clearer (and even simpler) to return the full value, not the naked one:

```
return s.Image.AddFromFile(filePathName, sumOfyOffsets, s.rowColCount, qtdCols, s.rowHeight, rectProp)
```

Reply:
Make sense, I will change it :D